tests/Behat/Resources/services/contexts/ui.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ This file is part of the Sylius package.
+ (c) Paweł Jędrzejewski
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+-->
```

Comment:
Why is this one compacted?

Reply:
We've got the same in https://github.com/Sylius/PriceHistoryPlugin/blob/main/tests/Behat/Resources/services/contexts/api.xml#L3-L8 and https://github.com/Sylius/PriceHistoryPlugin/blob/main/tests/Behat/Resources/services/contexts/setup.xml#L3-L8 and I relied on them. When I checked other places, we use the compressed version in the ApiBundle

Reply:
These docblocks should be fixed to have some empty lines like others